gcp_variant_transforms/vcf_to_bq.py

```diff
@@ -404,6 +404,23 @@ def _get_avro_root_path(beam_pipeline_options):
                                       _AVRO_FOLDER,
                                       google_cloud_options.job_name, '')
 
+def _rollback_newly_created_tables(append, base_table_name, suffixes=None):
+  if append:
+    logging.warning(
+        'Since tables were appended, added rows cannot be reverted. You can '
+        'utilize BigQuery snapshot decorators to recover your table up to 7 '
+        'days ago. For more information please refer to: '
+        'https://cloud.google.com/bigquery/table-decorators')
+  else:
+    logging.warning('Trying to revert as much as possible...')
+    suffixes = (suffixes.append(bigquery_util.SAMPLE_INFO_TABLE_SUFFIX)
+                if suffixes else [bigquery_util.SAMPLE_INFO_TABLE_SUFFIX])
+    for suffix in suffixes:
+      table_name = bigquery_util.compose_table_name(base_table_name, suffix)
+      if bigquery_util.delete_table(table_name) == 0:
+        logging.info('Table was successfully deleted: %s', table_name)
+      else:
+        logging.error('Failed to delete table: %s', table_name)
```

Comment:
I am thinking to move this function to `bigquery_util` module, what do you think?

Reply:
Sounds like a good idea:
1) vcf_to_bq is huge anyways, so any possible way to make it simpler is good in my books.
2) It's easier to unit test small functionalities in other files, so it would improve test coverage.

Reply:
Done.